### DISCUSSION TITLE
MPG->M4V, Captions

### DISCUSSION THREAD

**Discussion by TonyTTTT** [mifi/lossless-cut#1873] [General]
Is it possible for LosslessCut to copy the Closed Captions in an MPG file when converting to MP4?
The captions are listed as (ffprobe):
Stream #0:1[0x100]: Video: h264 (High) ([27][0][0][0] / 0x001B), yuv420p(tv, bt709, progressive), 1280x720 [SAR 1:1 DAR 16:9], Closed Captions, 59.94 fps, 59.94 tbr, 90k tbn

FWIW, Handbrake does copy them when converting.

**Comment by mifi** [comment]
tbh i don't know how closed captions work, but maybe choose "copy all mov metadata" or copy all metadata will help. or choose output format mov instead of mp4

**Comment by TonyTTTT** [comment]
The CC in the mpeg is a stream (see OP), so copy all metadata won't work (I did try it though)
LosslessCut discards the stream by default, so I tried enabling it as "captions", and tried MOV, didn't work either.

ffmpeg  -f lavfi -i movie="video.mpeg[out+subcc]" -map 0:1 Desktop/out.srt
works on some mpeg, but not all.  Not sure what Handbrake does, but it can convert the CC to mp4

**Comment by TonyTTTT** [comment]
When I checked the file in VLC and QT, there were no captions, but I just  transferred the file to PLEX, it recognizes the CC, so looks like LosslessCut does transfer the CC when converting from mpg->m4p

**Comment by mifi** [comment]
maybe you could share the data from Help -> report an error, then i can see how the file actually looks like

**Comment by TonyTTTT** [comment]
But it's not an error, as LosslessCut is copying the Closed Captions as PLEX can display them, Its VLC and QT that don't have the CC as a Subtitle choice
(I was reviewing the cut in VLC, and posted this question before I moved the file to PLEX)

**Reply by mifi** [reply]
unfortunately i'm not able to help when i don't know more about your file
